libtransact/src/lib.rs

```diff
@@ -147,6 +147,6 @@ extern crate log;
 #[cfg(feature = "sabre-compat")]
 #[macro_use]
 extern crate sabre_sdk;
-#[cfg(feature = "contract-archive")]
+#[cfg(any(feature = "contract-archive", feature = "workload-runner"))]
```

Comment:
Can probably just change this to `#[cfg(feature = "serde_derive")]` to avoid adding each feature that depends on that crate.

Reply:
fixed